odoo/tools/image.py

```diff
@@ -85,7 +85,7 @@ def __init__(self, source, verify_resolution=True):
             try:
                 self.image = Image.open(io.BytesIO(source))
             except (OSError, binascii.Error):
-                raise UserError(_("This file could not be decoded as an image file."))
+                raise UserError(_lt("This file could not be decoded as an image file."))
```

Comment:
This is not correct.  First, `_lt()` does not return a `str` but an object that returns the translated string.  And second, the string conversion that _actually_ performs the translation will be done somewhere in another stack frame that _possibly_ doesn't have a `self.env`, or that has a `self.env` _with another language in context_.

Reply:
Indeed it is not correct. Actually, `odoo.tools` should not try to translate anything except if it has explicitly some record or env. There are other usages like this and I just aligned this one: `_` is obviously incorrect, but `_lt` may resolve to something and if it does not find a language, it will log a warning.
Ideally, tools should raise some custom exceptions which could be caught and re-raised as translated exceptions in business code; otherwise `env` could be passed always as argument.

Do you prefer?
1. I leave as-is, until it is fixed if needed.
2. I add env to nearly all functions using `_lt` in tools.